lib/index.js

```diff
@@ -45,12 +46,21 @@ Elevio.prototype.identify = function(identify) {
   var name = identify.name();
   var email = identify.email();
   var plan = identify.proxy('traits.plan');
+  var traits = identify.traits();
+
+  var removeTraits = ['id', 'name', 'firstName', 'lastName', 'email'];
+
+  removeTraits.forEach(function(traitItem) {
+    if (traits && traitItem in traits) {
+      delete traits[traitItem];
```

Comment:
You'll need to use the `del` method from `obj-keys` here—some users send traits in camelcase.

https://github.com/segmentio/obj-case